Extension.php

```diff
@@ -98,7 +98,7 @@ public function initialize()
         // register route for all GET requests on '/example/url' that will be handled in this class ($this) in the 'routeExampleUrl' function.
         $this->app
             ->get('/example/url', array($this, 'routeExampleUrl'))
-            ->bind('example-url');
+            ->bind('example-url'); // route name, must be unique(!)
```

Comment:
It would be nice to be able to look into a crystal ball and see how many people are saved hours of head scratching and never know why thanks to you putting that comment in :+1: